README.md

```diff
@@ -1,2 +1,30 @@
-# Multilingual_NER
-Applying BERT to named entity recognition in English and Russian.
```

Comment:
Something more like the following:  "This repository applies BERT to named entity recognition in English and Russian in order to improve machine translation quality estimation between English-Russian sentence pairs. Named entities are a known challenge in machine translation, and this repository aims to quantify accuracy by comparing named entity recognition between Russian and English in translated sentence pairs."